models/apartment.rb

```diff
@@ -0,0 +1,11 @@
+class Apartment
+  attr_reader(:id)
```

Comment:
I really like that you made this an `attr_reader` as this sort of property should only be set at the database level. I was curious though why you decided against it for the tenant model.